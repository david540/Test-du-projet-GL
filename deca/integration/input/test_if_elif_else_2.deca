{
        if(1 < 2){
                println("coucou");
        }
        if(false || true){
            println("oui");
        }
        if(true || false){
            println("oui");
        }
        if(true || true){
            println("oui");
        }
        if(false || false){
            println("non");
        }
        if(false && true){
            println("non");
        }
        if(true && false){
            println("non");
        }
        if(true && true){
            println("oui");
        }
        if(false && false){
            println("non");
        }
}
