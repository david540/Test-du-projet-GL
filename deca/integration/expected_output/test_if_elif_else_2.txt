coucou
oui
oui
oui
oui
